### DISCUSSION TITLE
How does the printer tool work?

### DISCUSSION THREAD

**Discussion by timkolloch** [sysadminsmedia/homebox#388] [Q&A] [answered]
Hey guys!

I wanted to print labels for my items and fiddled around with the printer tool.
The QR codes are created at the bottom and everything is fine.
But now what?
Do I have to print the page as a pdf?
If I do that I got the print time and stuff like that at the top of the page which breaks the paddings etc. which is a bit annoying, or is this part of the fiddling around?

**Comment by tankerkiller125** [comment, marked as answer]
Make sure headers and margins are set to none in the browser print options/OS print options. They will add things that are not supposed to be there, and because we can't account for every browsers default options for margins and headers it would be unreasonable to attempt to take these things into account on the server side of things.

Maybe eventually we'll render PDFs instead which I don't believe have this issues, but that will be quite some time down the road before we can do that.

**Reply by timkolloch** [reply]
Oh I did not know that I could disable the margins when printing the page, thank you very much for the quick help, it works now!
